### ISSUE TITLE
Displaying attribute inheritance

### ISSUE THREAD

**Issue by Moult** [AECgeeks/ifc43-documentation-server#2]
The part of the spec I look at the most for any entity page is the attribute inheritance table. I realise this might be a little tricky with markdown, and probably needs a bit of fancy processing to merge all inherited attributes. Just starting the discussion in case there was any thoughts around this already.

**Comment by aothms** [comment]
Yeah this is definitely still on the table. But I first need to work through the generation of the pset xmls...
